webpages/reports/pubsbio.php

```diff
@@ -4,12 +4,13 @@
 $report['name'] = 'Pubs - Participant Bio and pubname';
 $report['multi'] = 'true';
 $report['output_filename'] = 'pubs_bios.csv';
-$report['description'] = 'Show the id, pubsname and bio for each participant who is on at least one scheduled, public session.';
-$report['categories'] = array(
+$report['description'] =
+    'Show the id, pubsname and bio for each participant who is on at least one scheduled, public session.';
+$report['categories'] = [
```

Comment:
Why was this changed from the array() syntax to the [] syntax? Please change it back for consistency.

Reply:
Okay, have changed it back. Though note that the $report['queries'] already used the newer [] syntax.